### ISSUE TITLE
Mistake in documentation

### ISSUE THREAD

**Issue by beemavishnu** [MicrosoftDocs/windows-driver-docs#830]
Hello,

1) Need to change "that is not in this list" to "that are in this list" in below statement.

WDI_TLV_DISALLOWED_BSSIDS_LIST X The list of BSSIDs that are not allowed to be used for association. If specified, the adapter must not associate to any AP that is not in this list. 

2) It would be good if all TLVs are in table format. Something like below link
https://docs.microsoft.com/en-us/windows-hardware/drivers/network/wdi-tlv-scan-mode

Thanks & Regards
Vishnu Beema

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 55ed9d58-53a0-482e-197b-2c6de9f06b0b
* Version Independent ID: 281142aa-df29-2103-6ad4-9efed33ee562
* Content: [WDI_TLV_CONNECT_PARAMETERS](https://docs.microsoft.com/en-us/windows-hardware/drivers/network/wdi-tlv-connect-parameters)
* Content Source: [windows-driver-docs-pr/network/wdi-tlv-connect-parameters.md](https://github.com/MicrosoftDocs/windows-driver-docs/blob/staging/windows-driver-docs-pr/network/wdi-tlv-connect-parameters.md)
* Product: **windows-hardware**
* GitHub Login: @windows-driver-content
* Microsoft Alias: **windowsdriverdev**

**Comment by EliotSeattle** [comment]
Thanks, @beemavishnu. @duncanmacmichael, looks like this was formatted as a table but something went wrong along the way to publishing. Can you make it purty? Might want to do a quick spot check of the other TLVs to see if it's more widespread than just the one.

**Comment by duncanmacmichael** [comment]
Hi @beemavishnu , thanks for finding this. It looks like the table was garbled during a content migration we did a while back and ended up slipping through the cracks. We usually do have the TLV values in table form, and I confirmed that this is the only one I know of by doing a random spot check of a dozen+ other TLVs. I've made the fix and the change should be live later today or tomorrow. Thanks for your interest in the docs!
